docs/docs/start-here.mdx

```diff
@@ -0,0 +1,207 @@
+    <ChatUserText>Indian food</ChatUserText>
+    <ChatBotText>in what price range?</ChatBotText>
+    <ChatUserText>actually no I want Italian. A cheap spot</ChatUserText>
+    <ChatBotText>and in which city?</ChatBotText>
+    <ChatUserText>Berlin</ChatUserText>
+    <ChatBotText>Here's a recommendation: ...</ChatBotText>    
+    </Chat>
+  </TabItem>
+  <TabItem value="interjection">
+    <Chat caption="user changes their mind">
+    <ChatUserText>Can you recommend somewhere to eat?</ChatUserText>
+    <ChatBotText>What kind of food are you looking for?</ChatBotText>
+    <ChatUserText>wait are you a bot?</ChatUserText>
+    <ChatBotText>I am a bot, powered by Rasa.</ChatBotText>
+    <ChatBotText>What kind of food are you looking for?</ChatBotText>    
+    </Chat>
+  </TabItem>  
+</Tabs>
+
+
+### Understanding flows
+
+Given the range of conversations this assistant can handle, you might expect the implementation to be complex.
+In fact, there are only two small files that provide Rasa what it needs. 
+
+The `data/flows/restaurants.yml` file defines the logic for this flow. In this example the logic is linear
+and walks the user through each of the steps in order. Each of the `question` steps fills the corresponding slot.
+
+```yaml
+flows:
+  recommend_restaurant:
+    description: This flow recommends a restaurant
+    steps:
+      - id: "0"
+        question: cuisine
+        skip_if_filled: true
+        next: "1"
+      - id: "1"
+        question: price_range
+        next: "2"
+      - id: "2"
+        question: part_of_town
+        next: "3"
+      - id: "3"
+        action: search_restaurants
+```
+
+To build more advanced flows, you can add conditional logic, link to other flows, and more. See [add link]
+
+In addition to the flows, the `domain.yml` file contains definitions of the slots and responses used in this flow. 
+
+### Understanding DM2
+
+Rasa uses a new approach to building AI assistants called DM2 (name TBD). 
+If you've built AI assistants before, you might look at your project and think that many things are missing.
+
+* There is no NLU data with intents and entities
+* There is no logic mapping an intent like "restaurant_search" to the start of the restaurant flow
+* There are no slot mappings
+* There is no explicit logic handling corrections, interruptions, or other unhappy paths.
+
+DM2 doesn't need any of these things to be able to handle the example conversations above. So, how does that work?
+
+Many developers are familiar with dialogue systems made up of separate NLU, dialogue, and NLG components
+(this is also how Rasa worked previously).
+
+In DM2, we have a different set of modules. 
+
+* The *conversation handling* component (name TBD) interprets the conversation so far and predicts 
+a series of commands to progress the state of the conversation.
+* The *business logic* component executes those commands and the logic of your flows. 
```

Comment:
is this the `FlowPolicy`? should we link to that page?

Reply:
This is minor, but I wonder if business logic is the best way to name this.